__tests__/app.test.js

```diff
@@ -134,6 +146,44 @@ describe('GET /api/articles?order', () => {
   });
 });
 
+describe('GET /api/articles?topic', () => {
+  test('200: Responds with an array of articles matching topic query', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(4);
+        articles.forEach((article) => {
+          expect(article.topic).toBe('mitch');
+        });
+      });
+  });
+  test('200: Responds with an empty array if topic exists, but no articles match topic query', () => {
+    return request(app)
+      .get('/api/articles?topic=paper')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(0);
+      });
+  });
+  test('200: Responds with an array of all articles if query is omitted', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+      });
```

Comment:
We already have code to test `GET /api/articles`! We don't need to retest the code here, we're introducing a `topic` query so we should only be testing the behavior of that.

Reply:
Understood, thank you! :saluting_face: